scripts/metrics/common/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Helper file for assisting in building workspaces and setting up LTT with the current Lumberyard environment.
```

Comment:
Each file contains this invalid docstring

Reply:
Fixed